src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltype/Example9.java

```diff
@@ -0,0 +1,31 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="IllegalType">
+      <property name="validateAbstractClassNames" value="true"/>
+      <property name="legalAbstractClassNames" value="AbstractList"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.illegaltype;
+
+import java.util.AbstractList;
+import java.util.AbstractSet;
+import java.util.ArrayList;
+import java.util.HashSet;
+
+// xdoc section -- start
+public class Example9 {
```

Comment:
You probably noticed that all examples are same class by structure 
https://checkstyle.org/checks/coding/illegaltype.html#Example8-code
And only trailing comments are changing locations, this help user quickly notice difference after additional Check property is used.

Please do same.